homeassistant/components/alarmdecoder/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "AlarmDecoder",
   "codeowners": [],
   "config_flow": true,
+  "integration_type": "device",
```

Comment:
By changing the integration type here, it is breaking hassfest.
Please ensure that you run `python3 -m script.hassfest`

Reply:
I did run this on my side, but I didn't see any files changed as a consequence of this.  Do you have any insight into which file I'm supposed to pick up?

Reply:
Any insight into what is tripping it up?

Reply:
Ah! indeed.
The issues seems to be that you didn't insert it in the right place.
It should be ordered and moved under "documentation"
Then script should run correctly.

Reply:
You actually have to run it twice in your case:
- once to sort the manifest (which will fix the incorrect order)
- then a second time to update the `homeassistant/generated/integrations.json` generated file

Reply:
Ah yes perfect.  That's it.  It was generated the second time and I see it now :)  Thanks.